packages/client/src/components/UploadForm.tsx

```diff
@@ -108,12 +121,26 @@ const updateFileStatus = (
 	};
 };
 
-const checkUrlValid = (url: string) => {
+const checkUrlInputValid = (input: MediaUrlInput): MediaUrlInput => {
```

Comment:
does this parameter need to change? it seems like the function is simpler if it continues to accept the url string to validate.